ext/fetch/22_body.js

```diff
@@ -393,7 +393,7 @@
         return webidl.converters["ArrayBufferView"](V, opts);
       }
     }
-    return webidl.converters["USVString"](V, opts);
+    return webidl.converters["DOMString"](V, opts);
```

Comment:
`BodyInit` is a WebIDL union type defined in the fetch specification. As I suggested in https://github.com/denoland/deno/pull/12168#discussion_r713215118, you should probably define a different WebIDL type (`BodyInit_DOMString` for example), to avoid confusion.

Reply:
@andreubotella, what would you suggest to do with the nullable converter `BodyInit?` ? Should I rename it too?

Reply:
That's right

Reply:
Renamed.